Source/ARTRest.h

```diff
@@ -17,6 +17,16 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+@interface ARTHttpMethods : NSObject
```

Comment:
Please could you explain your thinking behind introducing this class into the public interface?

Reply:
>In which case this PR seems like an enhancement to the existing API, but doesn't add any new functionality

Yes, exactly. It's just wrapper functions.

Reply:
>keep the existing interface or add a new one?

Removing existing one probably will be very painful for users...

Reply:
Yeah, we definitely can't remove the existing one.

Does that mean we can already mark #780 as closed? I think that I'd like to get @QuintinWillison's opinion on whether it's worth adding these extra convenience methods or not.